auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -126,24 +147,19 @@ interface Authenticator {
          *                     interceptors, read timeouts, base url etc...) that can be set for
          *                     authentication and making requests.
          */
-        fun create(serverConfig: ServerConfig): Authenticator {
-
+        @JvmName("create")
```

Comment:
This should also have `@JvmStatic` so it can be `Authenticator.create(serverConfig)` instead of `Authenticator.Companion.create(serverConfig)` for Java users.